draft-packages/form/KaizenDraft/Form/Primitives/TextArea/TextArea.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import { cleanup, fireEvent, render, screen } from "@testing-library/react"
+import * as React from "react"
+import { TextAreaProps } from "./TextArea"
+import { TextArea } from "."
+
+afterEach(cleanup)
```

Comment:
This is an FYI, don't worry about fixing this now because we can do a sweep, but apparently these `afterEach(cleanup)` calls are obsolete now: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup

Reply:
Good to know. When I was trying to write another test that fired a mock function it didn't actually clean it up and kept incrementing. So I changed it to this and it worked (but don't need it anymore as I had to scrap that test):

```
afterEach(() => {
  jest.resetAllMocks()
})
```